.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: php
+
+php:
+  - '5.5'
+  - '5.6'
+  - '7.0'
+
```

Comment:
add `%travis.append.env%` and use to append symfony specific conf

(same for other template files)